File: Android/tuilivekit/src/main/java/com/trtc/uikit/livekit/state/operation/BeautyState.java
package com.trtc.uikit.livekit.state.operation;

import com.trtc.tuikit.common.livedata.LiveData;

public class BeautyState {
    public LiveData<Integer> smoothLevel         = new LiveData<>(6);
    public LiveData<Integer> whitenessLevel      = new LiveData<>(6);
    public LiveData<Integer> ruddyLevel          = new LiveData<>(6);
    public LiveData<Boolean> glContextCreateFlag = new LiveData<>();


    public void reset() {
        smoothLevel.set(6);
        whitenessLevel.set(6);
        ruddyLevel.set(6);
        glContextCreateFlag.set(null);
    }
}

File: Flutter/livekit/lib/state/operation/beauty_state.dart
import 'package:flutter/material.dart';

class BeautyState {
  final ValueNotifier<int> smoothLevel = ValueNotifier<int>(6);
  final ValueNotifier<int> whitenessLevel = ValueNotifier<int>(6);
  final ValueNotifier<int> ruddyLevel = ValueNotifier<int>(6);
  final ValueNotifier<bool> enableBeauty = ValueNotifier(false);
  final ValueNotifier<int> xmagicType = ValueNotifier(0);

  void reset() {
    smoothLevel.value = 6;
    whitenessLevel.value = 6;
    ruddyLevel.value = 6;
  }
}
